packages/dos-donts-block/src/DosDontsBlock.tsx

```diff
@@ -200,23 +200,43 @@ export const DosDontsBlock: FC<BlockProps> = ({ appBridge }) => {
         setAndSaveItems(newItems);
     };
 
+    const onChangeLocalItem = (
+        itemId: string,
+        value: string | number | undefined,
+        type: 'title' | 'body' | 'type' | 'imageId'
```

Comment:
```suggestion
        type: types
```
add a new type for this union so we can get rid of the repetition

Reply:
Thanks, fixed and the other one as well.